colplus-admin/src/test/java/org/col/admin/task/importer/acef/AcefDateParserTest.java

```diff
@@ -1,66 +1,36 @@
 package org.col.admin.task.importer.acef;
 
-import org.col.parser.UnparsableException;
-import org.junit.Test;
-
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
 import java.time.LocalDate;
 import java.util.Optional;
-
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.fail;
+import org.col.parser.UnparsableException;
+import org.col.util.date.FuzzyDate;
 
 @SuppressWarnings("static-method")
 public class AcefDateParserTest {
-  AcefDateParser parser = AcefDateParser.PARSER;
 
-  @Test
-  public void testEmpty() throws Exception {
-    assertEquals(Optional.empty(), parser.parse(""));
-    assertEquals(Optional.empty(), parser.parse(null));
-    assertEquals(Optional.empty(), parser.parse(" "));
-    assertEquals(Optional.empty(), parser.parse("   "));
+  public void test1() throws UnparsableException {
+    Optional<FuzzyDate> fd = AcefDateParser.PARSER.parse("30-Sep-2011");
+    assertTrue("01", fd.isPresent());
+    assertEquals("02", LocalDate.of(2011, 9, 30));
   }
 
-  void assertUnparsable(String x) throws UnparsableException {
-    try {
-      Optional<?> val = parser.parse(x);
-      // we should never reach here
-      fail("Expected "+x+" to be unparsable but was "+ (val.isPresent() ? val.get() : "EMPTY"));
-    } catch (UnparsableException e) {
-      // expected
-    }
+  public void test2() throws UnparsableException {
+    Optional<FuzzyDate> fd = AcefDateParser.PARSER.parse("31-Jan-2017 18:49:35");
+    assertTrue("01", fd.isPresent());
+    assertEquals("02", LocalDate.of(2017, 1, 31));
   }
 
-  @Test
-  public void testUnparsable() throws Exception {
-    for (String x : new String[]{"321", "jan", "31-01"}) {
-      assertUnparsable(x);
-    }
+  public void test3() throws UnparsableException {
+    Optional<FuzzyDate> fd = AcefDateParser.PARSER.parse("Nov-2010");
+    assertTrue("01", fd.isPresent());
```

Comment:
what is "01" used for? Is it just identifying in logs where a test has failed? I don't find this very useful, rather confusing. Can we either use real messages or none?